vector/src/main/java/im/vector/ui/PasswordStrengthBar.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2019 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package im.vector.ui
+
+import android.content.Context
+import android.graphics.Color
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.LinearLayout
+import butterknife.BindView
+import butterknife.ButterKnife
+import im.vector.R
+
+/*
+A password strength bar custom widget
+ Strength is an Integer
+ # -1 No strength
+ # 0 Weak
+ # 1 Fair
+ # 2 Good
+ # 3 Strong
+ */
+class PasswordStrengthBar(context: Context, attrs: AttributeSet? = null) : LinearLayout(context, attrs) {
```

Comment:
You should use `@JvmOverload` for the constructor

Reply:
done